app/helpers/chat_channel_memberships_helper.rb

```diff
@@ -15,4 +15,30 @@ def formatted_membership_user(memberships)
       }
     end
   end
+
+  def membership_users(memberships)
```

Comment:
Where is this used? I can't seem to find it elsewhere in this PR.

Reply:
it is used in this file `app/views/chat_channel_memberships/chat_channel_info.json.jbuilder`. I was just shifting the common code under the one file.